testing/config/settings/base.py

```diff
@@ -91,3 +91,6 @@
 PROJECT_IMPORTS_LIST = ['url']
 
 PROJECT_VALUES_VALIDATION = True
+
+PROJECT_CONTACT = True
+PROJECT_CONTACT_RECIPIENTS = ['email@example.com']
```

Comment:
A nice-to-have feature request for this PR would be to have this setting to be editable in the admin UI..
That would open a whole other feature of having settings in the Django Models right?

Reply:
yes, lets not start this now. we would need a settings or config model and also some good way to cache the entries there. the nice thing about `django.conf.settings` is that its super fast.

also: settings by site? 🤯

Reply:
alright, yeah settings is nice and fast ;)
of course it would be by site and maybe role-based permissions, etc..